library/src/main/java/com/telefonica/mistica/sheet/Sheet.kt

```diff
@@ -204,6 +246,35 @@ private fun ListInformative.toView(context: Context): View =
         it.adapter = InformativeListAdapter(this.elements.mapToInformativeViewData())
     }
 
+private fun BottomActions.toView(context: Context, onSheetTapped: InternalOnSheetTapped, container: ViewGroup): View {
+    return LayoutInflater.from(context).inflate(R.layout.sheet_bottom_actions, container, false)
+        .also { view ->
+            val primaryButton = view.findViewById<Button>(R.id.sheet_action_primary_button)
+            val secondaryButton = view.findViewById<Button>(R.id.sheet_action_secondary_button)
+            val linkButton = view.findViewById<com.telefonica.mistica.button2.Button>(R.id.sheet_action_link_button)
+            primaryButton.text = this.primaryButton.title
+            primaryButton.setOnClickListener {
+                onSheetTapped.onTapped(this.id, "PRIMARY")
+            }
+            secondaryButton.visibility = this.secondaryButton?.let { button ->
+                secondaryButton?.text = this.secondaryButton?.title.orEmpty()
```

Comment:
`this.linkButton` is already checked and it is available as non null as `button`


```suggestion
                secondaryButton?.text = button.title
```

Reply:
Yup, that makes sense, updated.
Thanks!